client/src/components/TaskCard/TaskCard.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { PropTypes } from "prop-types";
+import SmallActionButton from "../SmallActionButton/SmallActionButton.js";
+import styles from "./TaskCard.module.css";
+
+const TaskCard = ({
+  title,
+  location,
+  startTime,
+  endTime,
+  isOngoing,
+  showDetailsDialog,
+}) => {
+  const [taskOngoing, setOngoing] = useState(isOngoing);
```

Comment:
This can be left as `True` at the beginning since a Task that's always created should always be completed. I don't think there should be a case where a task can start as completed

Reply:
This was intended to be set when loading task data from the database. When pulling data from the database, some of the tasks would have already been completed.

Reply:
Right that makes sense, good catch!